packages/cactus-plugin-ledger-connector-iroha/src/main/json/openapi.json

```diff
@@ -173,9 +173,659 @@
+              },
+              {
+                "$ref": "#/components/schemas/SetAccountDetailRequestParameters"
+              },
+              {
+                "$ref": "#/components/schemas/SetAccountQuorumRequestParameters"
+              },
+              {
+                "$ref": "#/components/schemas/SubtractAssetQuantityRequestParameters"
+              },
+              {
+                "$ref": "#/components/schemas/TransferAssetRequestParameters"
+              },
+              {
+                "$ref": "#/components/schemas/CompareAndSetAccountDetailRequestParameters"
+              },
+              {
+                "$ref": "#/components/schemas/GetAccountRequestParameters"
+              },
+              {
+                "$ref": "#/components/schemas/GetBlockRequestParameters"
+              },
+              {
+                "$ref": "#/components/schemas/GetSignatoriesRequestParameters"
+              },
+              {
+                "$ref": "#/components/schemas/GetTransactionsRequestParameters"
+              },
+              {
+                "$ref": "#/components/schemas/GetPendingTransactionsRequestParameters"
+              },
+              {
+                "$ref": "#/components/schemas/GetAccountTransactionsRequestParameters"
+              },
+              {
+                "$ref": "#/components/schemas/GetAccountAssetTransactionsRequestParameters"
+              },
+              {
+                "$ref": "#/components/schemas/GetAccountAssetsRequestParameters"
+              },
+              {
+                "$ref": "#/components/schemas/GetAccountDetailRequestParameters"
+              },
+              {
+                "$ref": "#/components/schemas/GetAssetInfoRequestParameters"
+              },
+              {
+                "$ref": "#/components/schemas/GetRolePermissionsRequestParameters"
+              },
+              {
+                "$ref": "#/components/schemas/GetEngineReceiptsRequestParameters"
+              }
+            ]
+          }
+        }
+      },
+      "AddAssetQuantityRequestParameters": {
+        "description": "The list of arguments to pass in to the transaction request to Add Asset Quantity.",
+        "type": "object",
+        "required": [
+          "assetId",
+          "amount"
+        ],
+        "properties": {
+          "assetId": {
+            "type": "string"
+          },
+          "amount": {
+            "type": "integer"
+          }
+        }
+      },
+      "AddPeerRequestParameters": {
+        "description": "The list of arguments to pass in to the transaction request to Add Peer.",
+        "type": "object",
+        "required": [
+          "address",
+          "peerKey"
+        ],
+        "properties": {
+          "address": {
+            "type": "string"
+          },
+          "peerKey":{
+            "type": "string"
+          },
+          "tlsCertificate":{
+            "type": "string"
+          }
```

Comment:
AddPeer should contain one more param:
https://iroha.readthedocs.io/en/develop/develop/api/commands.html#add-peer
`syncing_peer`

Reply:
The current cactus version doesn't support the syncing_peer field. So can we keep it optional until the cactus is updated with the latest version?